flixel/text/FlxText.hx

```diff
@@ -1271,6 +1271,9 @@ enum abstract FlxTextAlign(String) from String
 
 	public static function fromOpenFL(align:TextFormatAlign):FlxTextAlign
 	{
+		if (align == null)
+			return LEFT;
+		
 		return switch (align)
 		{
 			case TextFormatAlign.LEFT: LEFT;
```

Comment:
nit picky, but might as well use the switch
```suggestion
		return switch (align)
		{
			case null: LEFT;
			case TextFormatAlign.LEFT: LEFT;
```

Reply:
in fact why does `default: LEFT` not include null already?

Reply:
I didn't know `null` in the switch could work, I'll give it a try.

> in fact why does `default: LEFT` not include null already?

No clue, it only happens when compiling to HashLink.

Reply:
weird: https://try.haxe.org/#0eABce28
looks like null is being cast to int: 0

should add a comment saying this is needed on HL, please

Reply:
Ah I forgot to do that, adding it now.